packages/http-signature-utils/src/utils/jwk.ts

```diff
@@ -27,6 +27,9 @@ export const generateJwk = ({
   const jwk = createPublicKey(privateKey).export({
     format: 'jwk'
   })
+  if (jwk.x === undefined) {
```

Comment:
Could possibly add a separate test for this if you want to test for the specific error msg

Reply:
How do I spy on 
https://github.com/interledger/rafiki/blob/421d89ac7883cec8d0f9f5a0d31cda98b6a9a448/packages/http-signature-utils/src/utils/jwk.ts#L27-L29

Reply:
I tried
```ts
 test('throws if invalid public key', async (): Promise<void> => {
      jest
        .spyOn(KeyObject.prototype, 'export')
        .mockImplementationOnce(() => ({ x: undefined }))

      expect(() => generateJwk({ keyId: 'keyid' })).toThrow(
        'Failed to derive public key'
      )
    })
```
To no avail. I think we can leave it out.